yash-cli/tests/scripted_test/option-y.sh

```
@@ -96,3 +96,122 @@ __IN__
+echo "$-" | grep -q a
+__IN__
+
+test_x -e 0 'option before enabling the portable option is not checked' --allexport -o portable
+echo "$-" | grep -q a
+__IN__
+
+test_x -e 0 'shell invocation can turn the portable option off again' -o portable +o portable --allexport
+echo "$-" | grep -q a
+__IN__
+
+test_O -d -e 2 'set built-in rejects -o argument attached to the option name' -o portable
```

Shellcheck (0.11.0)

[info] Command appears to be unreachable. Check usage (or ignore if invoked indirectly).

(SC2317)